### PR TITLE
Fix/sync upload for 0proxy

### PR DESCRIPTION
**Pull request by cnlangzi**
- updated sdk to support to work on 0Proxy

### CODE REVIEW

**Comment by cnlangzi** [comment]
fixed https://github.com/0chain/roadmap/issues/44

File: zboxcore/sdk/live_upload_reader_youtubedl.go [hunk 1]
```diff
@@ -102,15 +101,15 @@ func (r *YoutubeDL) wait() {
 func (r *YoutubeDL) Close() error {
 	if r != nil {
 		if r.cmd != nil {
-			r.cmd.Process.Kill()
+			r.cmd.Process.Kill() //nolint
```
Comment (kushthedude) [inline zboxcore/sdk/live_upload_reader_youtubedl.go:104]: What does the lint complain over here?

**Review by kushthedude — APPROVED** [review]
looks good to me, just a small query

File: zboxcore/sdk/live_upload_reader_youtubedl.go (hunk 1 above)
Comment (cnlangzi) [reply zboxcore/sdk/live_upload_reader_youtubedl.go:104]: because there is an `error` output. but we don't need to check it there.

**Pull request merged**